deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1528,6 +1512,20 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
         navigationController?.present(alert, animated: true, completion: nil)
     }
 
+    private func showAppPicker() {
+        let appPicker = AppPickerViewController()
+        appPicker.delegate = self
+        let navigationController = UINavigationController(rootViewController: appPicker)
+        navigationController.isModalInPresentation = true
```

Comment:
I really dislike that you can't dismiss this interactively. It makes sense during download but not really any other time.

Reply:
you mean "swipe down to close"? idk, "files" and "gallery" have the same behavior, therefore i thought not too much about it. if it is an easy change, however, of couse, i would not be against having the same behavior as our contact or profile-switcher :)

Reply:
Hmmm, that is actually an unintended side effect of the first responder fixes I did and did not notice :/

Reply:
ah! :)